source/slang/slang-ir-uniform-usage.cpp

```
@@ -0,0 +1,162 @@
+
+static UInt _getUniformSize(IRTypeLayout* typeLayout)
+{
+    if (!typeLayout)
+        return 0;
+    auto sizeAttr = typeLayout->findSizeAttr(LayoutResourceKind::Uniform);
+    if (!sizeAttr)
+        return 0;
+    auto size = sizeAttr->getSize();
+    if (!size.isFinite())
+        return 0;
+    return UInt(size.getFiniteValue().getValidValue());
```

Cppcheck (2.20.0)

[error] There is an unknown macro here somewhere. Configuration is required. If FIDDLE is a macro then please configure it.

(unknownMacro)